library/packages/src/modules/PackageCallbacks.rb

```diff
@@ -402,6 +403,22 @@ def ProgressPackage(percent)
       true
     end
 
+    # Handle GPG check result (pkgGpgCheck)
+    #
+    # If insecure mode is set to '1', the check result is ignored. Otherwise, no decision is made.
+    #
+    # @param data [Hash] Output from `pkgGpgCheck` callback.
+    # @option data [Integer] "CheckPackageResult" Check result code according to libzypp.
+    # @option data [String]  "Package" Package's name.
+    # @option data [String]  "Localpath" Path to RPM file.
+    # @option data [String]  "RepoMediaUrl" Media URL.
+    # @return [String] "I" if the package should be accepted; otherwise
+    #   a blank string is returned (so no decision is made).
+    def pkg_gpg_check(data)
+      log.debug("pkgGpgCheck data: #{data}")
+      Linuxrc.InstallInf("Insecure") == "1" ? "I" : ""
```

Comment:
To avoid an [ugly error message](https://github.com/yast/yast-yast2/blob/c84f544a93f3706a00ad5d4cf841ab68fee8550f/library/general/src/modules/Linuxrc.rb#L55) I'd use
```ruby
(Stage.initial && Linuxrc.InstallInf("Insecure") == "1")
```

condition here.